packages/ERTP/src/issuer.js

```diff
@@ -114,6 +114,8 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
       });
     }
 
+    assert(purses.length === newPurseBalances.length, details``);
```

Comment:
Better to omit the details than provide an empty one, unless you want to suppress `assert`'s default 

```js
details`check failed`
```

Reply:
Yup, this was an accident